services/workflows-service/src/workflow/cron/ongoing-monitoring.cron.ts

```diff
@@ -0,0 +1,218 @@
+import { Injectable } from '@nestjs/common';
```

Comment:
There is relatively a lot of logic in this file, it should be tested with at least unit tests